WebRoot/css/zm.css

```diff
@@ -7139,6 +7139,27 @@ a.ZmLinkDisabled {
 	@ContactListPaginationLinkIcon@
 }
 
+/* Quick add contact dialog*/
+.ZmContactQuickAddDialog {
+	@ZmQuickAddDialog@
```

Comment:
Will work, but I have choose that name to reduce number of chars. As subsequent property names usually uses `ZmQuickAddDialog` as prefix. 

It just my preference, however I am open to change it if you feels so. 

Let me know. 

Thanks